### PR TITLE
Update manager mode

### PR DESCRIPTION
**Pull request by mamy-CS**
Addresses: https://github.com/spiffe/tornjak/issues/511 and other updates in manager mode

### CODE REVIEW

File: frontend/src/components/navbar.tsx [hunk 1]
```diff
@@ -89,9 +93,18 @@ class NavigationBar extends Component<NavigationBarProp, NavigationBarState> {
...
-    this.TornjakApi.populateLocalTornjakServerInfo(this.props.tornjakServerInfoUpdateFunc, this.props.tornjakMessageFunc);
-    this.TornjakApi.populateLocalTornjakDebugServerInfo(this.props.spireDebugServerInfoUpdateFunc, this.props.tornjakMessageFunc);
-    this.TornjakApi.populateServerInfo(this.props.globalTornjakServerInfo, this.props.serverInfoUpdateFunc);
+    if (IsManager) {
+      if (this.props.globalServerSelected !== "" && (this.props.globalErrorMessage === "OK" || this.props.globalErrorMessage === "")) {
+        // this.TornjakApi.populateAgentsUpdate(this.props.globalServerSelected, this.props.agentsListUpdateFunc, this.props.tornjakMessageFunc)
+        // this.TornjakApi.populateEntriesUpdate(this.props.globalServerSelected, this.props.entriesListUpdateFunc, this.props.tornjakMessageFunc)
+        // this.TornjakApi.refreshSelectorsState(this.props.globalServerSelected, this.props.agentworkloadSelectorInfoFunc);
+        // this.setState({ selectedServer: this.props.globalServerSelected });
+      }
```
Comment (maia-iyer) [inline frontend/src/components/navbar.tsx:102]: might this be dead code?

File: go.mod [hunk 2]
```diff
@@ -2,6 +2,8 @@ module github.com/spiffe/tornjak
 
 go 1.22
 
+toolchain go1.22.2
```
Comment (maia-iyer) [inline go.mod:5]: unsure if this is necessary? i'm curious was there an issue that needed this change?

Comment (mamy-CS) [reply go.mod:5]: Yes I was getting and that solved it. 
```
go run cmd/manager/main.go 
go: downloading go1.22 (darwin/arm64)
go: download go1.22 for darwin/arm64: toolchain not available
```

File: frontend/src/components/navbar.tsx (hunk 1 above)
Comment (mamy-CS) [reply frontend/src/components/navbar.tsx:102]: I left that on purpose, will clean that up on the next pr when I update debugserver info.

**Pull request merged**